go.mod

```diff
@@ -1,6 +1,6 @@
-module github.com/urfave/cli/v2
+module github.com/urfave/cli/v3
 
-go 1.18
+go 1.19
```

Comment:
We are targeting min 1.19 ?

Reply:
I don't remember why I did this 😐 I agree this value should be `1.18`